front/nuxt.config.js

```diff
@@ -143,16 +145,27 @@ module.exports = {
       : undefined,
     cache: true,
     modern: true,
+    sourceMap: true,
+    plugins: [
+      new SentryPlugin({
```

Comment:
its not needed. its already in the nuxt/sentry module
https://github.com/nuxt-community/sentry-module/blob/8d8c60697f8296ad251241813f81895ec5503c5a/lib/module.js#L182
need to set publishRelease option

Reply:
fixed. 

removed "dsn" because it's set already by default